docs/how-to-use-the-playground.md

```diff
@@ -44,145 +42,71 @@ trino@d2bbfccc7432:/$ trino
 
 ```SQL
-WITH totalsales AS (
-  SELECT
-    employee_id,
-    SUM(total_amount) AS sales_amount
-  FROM "metalake_demo.catalog_hive".sales.sales
-  GROUP BY
-    employee_id
-), rankedemployees AS (
-  SELECT
-    employee_id,
-    sales_amount,
-    RANK() OVER (ORDER BY sales_amount DESC) AS sales_rank
-  FROM totalsales
-)
-SELECT
-  e.employee_id,
-  given_name,
-  family_name,
-  job_title,
-  sales_amount
-FROM rankedemployees AS r
-JOIN "metalake_demo.catalog_postgres".hr.employees AS e
-  ON r.employee_id = e.employee_id
-WHERE
-  sales_rank = 1;
+SELECT given_name, family_name, job_title, sum(total_amount) AS total_sales
+FROM "metalake_demo.catalog_hive".sales.sales as s,
+  "metalake_demo.catalog_postgres".hr.employees AS e
+where s.employee_id = e.employee_id
+GROUP BY given_name, family_name, job_title
+ORDER BY total_sales DESC
+LIMIT 1;
 ```
 
-If you want to know top 10 customers who bought the most by state.
-You run the SQL.
+If you want to know top customers who bought the most by state, you can run this SQL.
 
 ```SQL
-WITH customersales AS (
-    SELECT
-        "metalake_demo.catalog_hive".sales.customers.customer_id,
-        customer_name,
-        customer_email,
-        location AS state,
-        SUM(total_amount) AS total_spent
-    FROM "metalake_demo.catalog_hive".sales.sales
-             JOIN "metalake_demo.catalog_hive".sales.customers
-                  ON "metalake_demo.catalog_hive".sales.sales.customer_id = "metalake_demo.catalog_hive".sales.customers.customer_id
-             JOIN "metalake_demo.catalog_hive".sales.stores
-                  ON "metalake_demo.catalog_hive".sales.sales.store_id = "metalake_demo.catalog_hive".sales.stores.store_id
-    GROUP BY
-        "metalake_demo.catalog_hive".sales.customers.customer_id,
-        customer_name,
-        customer_email,
-        location
-), rankedcustomersales AS (
-    SELECT
-        customer_id,
-        customer_name,
-        customer_email,
-        state,
-        total_spent,
-        RANK() OVER (PARTITION BY state ORDER BY total_spent DESC) AS customer_rank
-    FROM customersales
-)
-SELECT
-    customer_id,
-    customer_name,
-    customer_email,
-    state,
-    total_spent
-FROM rankedcustomersales
-WHERE
-    customer_rank <= 10
-ORDER BY
-    state,
-    customer_rank;
+SELECT customer_name, location, SUM(total_amount) AS total_spent
+FROM "metalake_demo.catalog_hive".sales.sales AS s,
+  "metalake_demo.catalog_hive".sales.stores AS l,
+  "metalake_demo.catalog_hive".sales.customers AS c
+WHERE s.store_id = l.store_id AND s.customer_id = c.customer_id
+GROUP BY location, customer_name
+ORDER BY location, SUM(total_amount) DESC;
 ```
 
-If you want to know that employees average performance rating and total sales.
-You run the SQL.
+If you want to know the employee's average performance rating and total sales, you can run this SQL.
 
 ```SQL
-set session allow_pushdown_into_connectors=false;
```

Comment:
Why do we remove this?

Reply:
Because that bug has been fixed right?

Reply:
see https://github.com/datastrato/gravitino/issues/1154

Reply:
> Because that bug has been fixed right?

But we haven't release a new version.

Reply:
The playground doesn't use the 0.3 release AFAIK.

Reply:
0.3.1 version will include this fix. Playground use 0.3.0 release now.

Reply:
The docker containers have been updated many times and have not followed our released versions. Are you saying this has changed?

Reply:
As an aside it would be great to include version information on what is in each container here https://hub.docker.com/u/datastrato

Reply:
The tag of Docker image include the version information.

Reply:
> The docker containers have been updated many times and have not followed our released versions. Are you saying this has changed?

After we released a new version, we will public the Docker image of this version.

Reply:
I've added it to both the SQL statements that needed it - note a user may not run those SQL statements in sequence.